README.md

```diff
@@ -5,11 +5,11 @@
 <!-- [![Coverage Status](https://codecov.io/gh/secondmind-labs/GPflux/branch/develop/graph/badge.svg?token=<token>)](https://codecov.io/gh/secondmind-labs/GPflux) -->
 [![Quality checks and Tests](https://github.com/secondmind-labs/GPflux/actions/workflows/quality-check.yaml/badge.svg)](https://github.com/secondmind-labs/GPflux/actions/workflows/quality-check.yaml)
 [![Docs build](https://github.com/secondmind-labs/GPflux/actions/workflows/deploy.yaml/badge.svg)](https://github.com/secondmind-labs/GPflux/actions/workflows/deploy.yaml)
-[![Slack Status](https://img.shields.io/badge/slack-GPflux-green.svg?logo=Slack)](https://join.slack.com/t/secondmind-labs/shared_invite/zt-oevvuosg-HMJeTtIIQUkNtpBu40wKvg)
```

Comment:
The slack badge didn't render on the pypi page, so i removed it here and added it as a plain link below.

Reply:
it seems to render fine for GPflow... any thoughts why it might be different?